src/components/LogoutButton/LogoutButtonComponent.js

```diff
@@ -1,12 +1,15 @@
 import React, { Component } from 'react';
-import { StyledLogoutButton } from 'components/LogoutButton/LogoutButtonStyles';
+// import { StyledLogoutButton } from 'components/LogoutButton/LogoutButtonStyles';
+import Button from '@material-ui/core/Button';
 
 export class LogoutButtonComponent extends Component {
   render() {
     return (
-      <StyledLogoutButton onClick={this.props.logout}>
+      <Button
+        style={{ color: '#E4E3E4', margin: '1rem' }}
```

Comment:
Is it a styling to implement the new app design or just your custom design to make it prettier?
@TomaTomoe pls review

Reply:
Just some fast styles to see how it can look. Anybody are welcome to fix it or change styles completely